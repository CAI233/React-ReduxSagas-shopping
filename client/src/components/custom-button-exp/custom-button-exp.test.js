import React from 'react';
import { shallow } from 'enzyme';
import CustomButton from './custom-button-exp.component';

describe(" 测试公共按钮 ",()=>{
    let wrapper,mockProps;
    beforeEach(()=>{
        mockProps = {};
        wrapper = shallow( <CustomButton { ...mockProps } /> );
    });
    it("CustomButton Snapshot",()=>{
        expect(wrapper).toMatchSnapshot();
    });
});