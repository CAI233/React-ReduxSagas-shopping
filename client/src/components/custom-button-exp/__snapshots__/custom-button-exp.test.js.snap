// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 测试公共按钮  CustomButton Snapshot 1`] = `ShallowWrapper {}`;

exports[` 测试单个商品显示  CollectionView Snapshot 1`] = `undefined`;

exports[` 测试单个商品显示  CustomButton Snapshot 1`] = `ShallowWrapper {}`;
